skimage/feature/blob.py

```diff
@@ -323,7 +323,7 @@ def blob_dog(image, min_sigma=1, max_sigma=50, sigma_ratio=1.6, threshold=0.5,
     -----
     The radius of each blob is approximately :math:`\sqrt{2}\sigma` for
     a 2-D image and :math:`\sqrt{3}\sigma` for a 3-D image.
-    """
+    """  # noqa
```

Comment:
I couldn't determine why this comment and the ones below were added here. Could you quickly elaborate? Otherwise looks good! :+1:

Reply:
If you run flake8 on the file from the command line it warns about line length because of long URLs in the References. This just avoids that warning.

It is not strictly necessary to pass on CI because I think it does not lint the legacy code, only changed lines.

Reply:
Alright, just wanted to understand why the linter was silenced there. Thanks for the explanation. 

Because it covers the entire docsting, maybe we could use a more specific error code. [I think it's E501?](https://pep8.readthedocs.io/en/latest/intro.html#error-codes)

```suggestion
    """  # noqa: E501
```

Can't test this suggestion right now so I won't commit it.